microprofile-health/src/main/java/org/jboss/eap/qe/microprofile/health/junit/ManualTests.java

```diff
@@ -1,4 +1,4 @@
-package org.jboss.eap.qe.microprofile.health;
+package org.jboss.eap.qe.microprofile.health.junit;
```

Comment:
Why the package name change?

Reply:
I find it appropriate to have such marker interface in a dedicated package, since those are only functinal to JUnit. Additionally, I moved these annotations from the `test` sources to the `main` ones, which allows for using them in other modules as well, see e.g.:  `DockerRequiredTests`.